hack/convert-output-to-tsv.py

```diff
@@ -36,34 +38,64 @@ def remove_prefix(text, prefix):
 
 gcs = GCSFileSystem()
 
+# create a service object to handle all aspects of generating a primary key
+@dataclass
+class PrimaryKeyGenerator:
+    table_name: str
+    pk_name: str = field(init=False)
+    firecloud: bool
+
+    # this will calculate pk_name during init
+    def __post_init__(self):
+        # most tables should have "dog_id" as a key
+        if self.table_name in {"hles_dog", "hles_cancer_condition", "hles_health_condition", "environment", "cslb"}:
+            self.pk_name = 'dog_id'
+        # owner table is linked to hles_dog via "owner_id"
+        elif self.table_name == 'hles_owner':
+            self.pk_name = 'owner_id'
+        else:
+            raise ValueError(f"Unrecognized table: {self.table_name}")
+
+    # this will return the firecloud entity names
+    def generate_entity_name(self):
+        if self.firecloud:
+            return f"{PRIMARY_KEY_PREFIX}:{self.table_name}_id"
+        return self.pk_name
+
+    # this will return row[entity_name]
+    def generate_primary_key(self, row):
+        # normal processing of IDs - PK is stored as PK
+        if not self.firecloud:
+            return row.pop(self.pk_name)
+        # generate ids for Firecloud compatibility
+        # hles_health_condition: read + copy dog_id and hs_condition and hs_condition_is_congenital
+        # concatenate and write out as entity_name, keep original PK
+        if self.table_name == "hles_health_condition":
+            # code to generate the value of primary key should be pulled out
+            # grab the congenital flag and convert to int
+            congenital_flag = row.get('hs_condition_is_congenital')
+            try:
+                congenital_flag = int(congenital_flag)
+            except TypeError:
+                log.info(f"Error, 'hs_condition_is_congenital' is not populated in {self.table_name}")
```

Comment:
nit: this should be either a `warning` or an `error` instead of an `info`, since it's an error message